src/atoms/link/Link.js

```diff
@@ -1,2 +1,48 @@
 // @vue/component
-export default {}
+import getClass from '../../../utils/helpers/get-class.js'
+
+export default {
+  mixins: [ getClass ],
```

Comment:
`  5:12  error  There should be no space after '['   array-bracket-spacing
  5:21  error  There should be no space before ']'  array-bracket-spacing`